app/src/main/java/com/teamagam/gimelgimel/app/map/view/MapEntityDetailsFragment.java

```diff
@@ -0,0 +1,70 @@
+package com.teamagam.gimelgimel.app.map.view;
+
+import android.content.Context;
+import android.databinding.ViewDataBinding;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import com.teamagam.gimelgimel.R;
+import com.teamagam.gimelgimel.app.common.base.view.fragments.BaseDataFragment;
+import com.teamagam.gimelgimel.app.mainActivity.view.MainActivity;
+import com.teamagam.gimelgimel.app.map.viewModel.MapEntityDetailsViewModel;
+import com.teamagam.gimelgimel.databinding.FragmentMapEntityDetailsBinding;
+
+import javax.inject.Inject;
+
+import butterknife.BindView;
+
+
+public class MapEntityDetailsFragment extends BaseDataFragment<MapEntityDetailsViewModel> {
+
+    public static final String NAME_KEY = "name";
+    public static final String DESCRIPTION_KEY = "description";
+
+    @BindView(R.id.map_entity_name)
+    TextView mEntityName;
+
+    @BindView(R.id.map_entity_description)
+    TextView mEntityDescription;
+
+    @Inject
+    MapEntityDetailsViewModel mViewModel;
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        View view = super.onCreateView(inflater, container, savedInstanceState);
+        Bundle args = getArguments();
+        if (args != null) {
+            mEntityName.setText(args.getString(NAME_KEY));
```

Comment:
I know this class isn't used yet, but shouldn't the values for the TextViews should be coming from this view's ViewModel ?
This bundling stuff is old-school Android